### ISSUE TITLE
I can't play this one specific YouTube video downloaded with yt-dl.

### ISSUE THREAD

**Issue by lucmp** [ytdl-org/youtube-dl#28653]
## Checklist
- [x ] I'm reporting a broken site support issue
- [x ] I've verified that I'm running youtube-dl version **2021.04.01**
- [x ] I've checked that all provided URLs are alive and playable in a browser
- [x ] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x ] I've searched the bugtracker for similar bug reports including closed ones
- [x ] I've read bugs section in FAQ

There is this video I am trying to download. YouTube videos always work fine for me, only this one fails. It seems to download correctly, but then I can't play it.

Link to the video: https://youtube.com/watch?v=J2NShmv8K-0

Error output:
------------------
$ mpv AMANDA\ KLEIN\ FICA\ SEM\ CHÃO\ AO\ CONFRONTAR\ MÉDICO\ E\ ELE\ DEVOLVER\ NA\ MESMA\ MOEDA-J2NShmv8K-0.mkv 
Playing: AMANDA KLEIN FICA SEM CHÃO AO CONFRONTAR MÉDICO E ELE DEVOLVER NA MESMA MOEDA-J2NShmv8K-0.mkv
[mkv] Unknown/unsupported CodecID (V_AV1) or missing/bad CodecPrivate data (track 1).
 (+) Video --vid=1 (*) ( 1280x720 30.000fps)
 (+) Audio --aid=1 --alang=eng (*) (opus 2ch 48000Hz)
Failed to initialize a video decoder for codec ''.
Video: no video
Exiting... (Errors when loading file)
------------------

It only happens to that one video. I can download just about any other YouTube video.

I also have many .mkv videos that I can play normally.

I tried it twice and it failed. So I checked for updates and installed the new version, 
$ youtube-dl --version
2021.04.01

Then I tried a third time, and the video still doesn't work.

Link to video: https://youtube.com/watch?v=J2NShmv8K-0

**Comment by rautamiekka** [comment]
Plays just fine in VLC: https://gofile.io/d/Wjk0Ab

^ The VLC debug messages also said it's an AV1, using the DAV1D 0.7.1 decoder.

Also, fill the template !

```
# youtube-dl --ignore-config --verbose --verbose --ignore-errors --no-mark-watched --format "bestvideo+bestaudio/best" --audio-format best --audio-quality 0 --ffmpeg-location "C:\Users\Public\Downloads\ffmpeg\bin" --no-playlist --output "I:\SEAGATE_2TB_0\_DOWNLOADS_\_websites_\com\youtube/%(uploader_id)s"/"%(title)s %(resolution)s #%(id)s#.%(ext)s" --console-title --no-mtime --retries infinite --continue --no-overwrites --fragment-retries infinite "https://youtube.com/watch?v=J2NShmv8K-0"
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--ignore-config', '--verbose', '--verbose', '--ignore-errors', '--no-mark-watched', '--format', 'bestvideo+bestaudio/best', '--audio-format', 'best', '--audio-quality', '0', '--ffmpeg-location', 'C:\\Users\\Public\\Downloads\\ffmpeg\\bin', '--no-playlist', '--output', 'I:\\SEAGATE_2TB_0\\_DOWNLOADS_\\_websites_\\com\\youtube/%(uploader_id)s/%(title)s %(resolution)s #%(id)s#.%(ext)s', '--console-title', '--no-mtime', '--retries', 'infinite', '--continue', '--no-overwrites', '--fragment-retries', 'infinite', 'https://youtube.com/watch?v=J2NShmv8K-0']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, pref cp1252
[debug] youtube-dl version 2021.04.01
[debug] Python version 3.8.8 (CPython) - Windows-10-10.0.19041-SP0
[debug] exe versions: ffmpeg 4.3.1-2021-01-26-full_build-www.gyan.dev, ffprobe 4.3.1-2021-01-26-full_build-www.gyan.dev
[debug] Proxy map: {}
[youtube] J2NShmv8K-0: Downloading webpage
[youtube] Downloading just video J2NShmv8K-0 because of --no-playlist
WARNING: Requested formats are incompatible for merge and will be merged into mkv.
[debug] Invoking downloader on 'https://r2---sn-qo5-2vgz.googlevideo.com/videoplayback?expire=1617510900&ei=lO1oYIPWLYvvyQXApJXABA&ip=2001%3A14ba%3A23e8%3A8f00%3A3c4a%3Aa604%3A5c74%3A996d&id=o-AAgIwr6aVsXYKf6bVYGA-agQJooD5Y9yIxvQ5dIr9Iob&itag=398&aitags=133%2C134%2C135%2C136%2C160%2C242%2C243%2C244%2C247%2C278%2C394%2C395%2C396%2C397%2C398&source=youtube&requiressl=yes&mh=TA&mm=31%2C29&mn=sn-qo5-2vgz%2Csn-5goeen7y&ms=au%2Crdu&mv=m&mvi=2&pl=40&initcwndbps=1482500&vprv=1&mime=video%2Fmp4&ns=j7k0oguO0gzS3tZHuxpK1pIF&gir=yes&clen=21760041&dur=569.066&lmt=1617464670653289&mt=1617488949&fvip=2&keepalive=yes&fexp=24001373%2C24007246&c=WEB&txp=5431432&n=Jd6QiopPorbqE_nB-&sparams=expire%2Cei%2Cip%2Cid%2Caitags%2Csource%2Crequiressl%2Cvprv%2Cmime%2Cns%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRQIhANY3oZ97ytaaGHzRZeCmYBpavDsm0o0flS7rS5arm7Q9AiBO8vkjk1Q8ohfvKvGXllaX_WK3Su0op7mjogEfemPyCw%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgTm7YdF3gYNscl6DCQwUTk4_-Mj2gIZ0P-ZZ4np7qLgkCIQDkAKUXI9XrBaRBtDwCcnO3LDOcHihaitFVNkVua0XKKA%3D%3D'
[download] Destination: I:\SEAGATE_2TB_0\_DOWNLOADS_\_websites_\com\youtube\UC54x4lI4tle1mTHh5mQkVGA\AMANDA KLEIN FICA SEM CHÃO AO CONFRONTAR MÉDICO E ELE DEVOLVER NA MESMA MOEDA 1280x720 #J2NShmv8K-0#.f398.mp4
[download] 100% of 20.75MiB in 06:01
[debug] Invoking downloader on 'https://r2---sn-qo5-2vgz.googlevideo.com/videoplayback?expire=1617510900&ei=lO1oYIPWLYvvyQXApJXABA&ip=2001%3A14ba%3A23e8%3A8f00%3A3c4a%3Aa604%3A5c74%3A996d&id=o-AAgIwr6aVsXYKf6bVYGA-agQJooD5Y9yIxvQ5dIr9Iob&itag=251&source=youtube&requiressl=yes&mh=TA&mm=31%2C29&mn=sn-qo5-2vgz%2Csn-5goeen7y&ms=au%2Crdu&mv=m&mvi=2&pl=40&initcwndbps=1482500&vprv=1&mime=audio%2Fwebm&ns=j7k0oguO0gzS3tZHuxpK1pIF&gir=yes&clen=9484427&dur=569.101&lmt=1617459627355574&mt=1617488949&fvip=2&keepalive=yes&fexp=24001373%2C24007246&c=WEB&txp=5431432&n=Jd6QiopPorbqE_nB-&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cvprv%2Cmime%2Cns%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRQIhALbyS4bO4nakRzn9bGOgXQJQCnqgOm2nLBTPUOeQU5DFAiA2R6Bkp_vQxX-YvXBJ1VBXm9WdnDI-j-2ASqqPQgzvNw%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgTm7YdF3gYNscl6DCQwUTk4_-Mj2gIZ0P-ZZ4np7qLgkCIQDkAKUXI9XrBaRBtDwCcnO3LDOcHihaitFVNkVua0XKKA%3D%3D'
[download] Destination: I:\SEAGATE_2TB_0\_DOWNLOADS_\_websites_\com\youtube\UC54x4lI4tle1mTHh5mQkVGA\AMANDA KLEIN FICA SEM CHÃO AO CONFRONTAR MÉDICO E ELE DEVOLVER NA MESMA MOEDA NA #J2NShmv8K-0#.f251.webm
[download] 100% of 9.05MiB in 02:37
[ffmpeg] Merging formats into "I:\SEAGATE_2TB_0\_DOWNLOADS_\_websites_\com\youtube\UC54x4lI4tle1mTHh5mQkVGA\AMANDA KLEIN FICA SEM CHÃO AO CONFRONTAR MÉDICO E ELE DEVOLVER NA MESMA MOEDA 1280x720 #J2NShmv8K-0#.mkv"
[debug] ffmpeg command line: "C:\Users\Public\Downloads\ffmpeg\bin\ffmpeg" -y -loglevel "repeat+info" -i "file:I:\SEAGATE_2TB_0\_DOWNLOADS_\_websites_\com\youtube\UC54x4lI4tle1mTHh5mQkVGA\AMANDA KLEIN FICA SEM CHÃO AO CONFRONTAR MÉDICO E ELE DEVOLVER NA MESMA MOEDA 1280x720 #J2NShmv8K-0#.f398.mp4" -i "file:I:\SEAGATE_2TB_0\_DOWNLOADS_\_websites_\com\youtube\UC54x4lI4tle1mTHh5mQkVGA\AMANDA KLEIN FICA SEM CHÃO AO CONFRONTAR MÉDICO E ELE DEVOLVER NA MESMA MOEDA NA #J2NShmv8K-0#.f251.webm" -c copy -map "0:v:0" -map "1:a:0" "file:I:\SEAGATE_2TB_0\_DOWNLOADS_\_websites_\com\youtube\UC54x4lI4tle1mTHh5mQkVGA\AMANDA KLEIN FICA SEM CHÃO AO CONFRONTAR MÉDICO E ELE DEVOLVER NA MESMA MOEDA 1280x720 #J2NShmv8K-0#.temp.mkv"
Deleting original file I:\SEAGATE_2TB_0\_DOWNLOADS_\_websites_\com\youtube\UC54x4lI4tle1mTHh5mQkVGA\AMANDA KLEIN FICA SEM CHÃO AO CONFRONTAR MÉDICO E ELE DEVOLVER NA MESMA MOEDA 1280x720 #J2NShmv8K-0#.f398.mp4 (pass -k to keep)
Deleting original file I:\SEAGATE_2TB_0\_DOWNLOADS_\_websites_\com\youtube\UC54x4lI4tle1mTHh5mQkVGA\AMANDA KLEIN FICA SEM CHÃO AO CONFRONTAR MÉDICO E ELE DEVOLVER NA MESMA MOEDA NA #J2NShmv8K-0#.f251.webm (pass -k to keep)
```

**Comment by kikuyan** [comment]
No problem with my mpv.
Probably a problem with your ffmpeg or mpv.

**Comment by october262** [comment]
no problem playing  the video on my end, might try youtube-dl -f mp4  https://youtube.com/watch?v=J2NShmv8K-0

**Comment by lucmp** [comment]
Sorry if this was some kind of false alarm to anyone. But like I said, I have been able to download videos from YouTube almost every day, I downloaded others on that same day and only that one video could not be played on my box and, well, youtube-dl is the software that is generating it (it downloads then converts). I also tried mplayer with the exact same result. The '-f mp4' option is good enough a workaround for me. Thank you for the attention.
